src/settings.py

```diff
@@ -54,11 +54,13 @@ class GlobalSettings:
     behavior_config_file = os.path.join(  # relative to igibson.root_path
         "examples",
         "configs",
-        # "njk_re-shelving_library_books_full_obs.yaml",
-        "njk_sorting_books_full_obs.yaml")
+        "wbm3_modifiable_full_obs.yaml",
+    )
     behavior_mode = "headless"  # headless, pbgui, iggui
     behavior_action_timestep = 1.0 / 10.0
     behavior_physics_timestep = 1.0 / 120.0
+    behavior_task_name = "re-shelving_library_books"
```

Comment:
Would it be easy/possible to add a comment describing what the options are for both of these settings? Maybe a link to a yaml file which defines all the options?